src/services/videoconference/logic/bbb.js

```diff
@@ -0,0 +1,18 @@
+const { server: createBBB } = require('bbb-promise');
+const { GETAction: get } = require('bbb-promise/lib/util');
+
+function bbb(host, salt) {
+	const api = createBBB(host, salt);
+
+	// Extend API wrapper to manage BBB WebHooks.
+	// http://docs.bigbluebutton.org/dev/webhooks.html
+	api.webhooks = {
+		create: (params) => get(host, salt, 'hooks/create', params),
+		destroy: (params) => get(host, salt, 'hooks/destroy', params),
+		list: (params = {}) => get(host, salt, 'hooks/list', params),
+	};
+
+	return api;
+}
```

Comment:
We ended up not needing the [BBB "webhooks" app](http://docs.bigbluebutton.org/dev/webhooks.html) in the end, as recordings and recording-ready-callbacks are already supported by the BBB core. This code is small & tested now, but if you prefer, we can drop it. Not sure if someone else has a use for the more advanced BBB webhooks? 🙂